### PR TITLE
#T-234: Product delete

### LINKED ISSUE #234: Product delete (UB3)
Image

### PR DESCRIPTION
**Pull request by michalpulpan**
closes #234 

 

https://github.com/ecoseller/ecoseller/assets/34132752/447affc9-9cb7-4468-b304-7dfe06792c4c

@VL-CZ before deleting an object in `EditableContentWrapper`, try to call `setPreventNavigation(false);` first

### CODE REVIEW

File: src/backend/core/country/urls.py
Comment (VL-CZ) [inline src/backend/core/country/urls.py:1]: It seems that you also included `country` API to this PR.
Is it correct?

File: src/backend/docker-compose.env [hunk 1]
```diff
@@ -13,4 +13,5 @@ REDIS_QUEUE_LOCATION=redis
 
 PYTHONUNBUFFERED=1
 
-RS_URL="http://recommender_system:8086"
\ No newline at end of file
+RS_URL="http://recommender_system:8086"
+RS_ENABLED="FALSE"
```
Comment (VL-CZ) [inline src/backend/docker-compose.env:17]: shouldn't be commited IMO 😄

File: src/backend/core/country/urls.py
Comment (michalpulpan) [reply src/backend/core/country/urls.py:1]: 💩 , that's not correct

File: src/backend/docker-compose.env (hunk 1 above)
Comment (michalpulpan) [reply src/backend/docker-compose.env:17]: Yeah, it shouldn't 😂

**Pull request closed without merging**